src/ucs/datastruct/bitmap.h

```diff
@@ -39,14 +39,23 @@ typedef uint64_t ucs_bitmap_word_t;
 
 
 /**
- * Word index of a bit in bitmap
+ * Word index of a bit in bitmap. Assert the bitmap is big enough
  *
+ * @param _bitmap    Index of this bit relative to the bitmap
  * @param _bit_index Index of this bit relative to the bitmap
  *
  * @return Index of the word this bit belongs to
  */
-#define UCS_BITMAP_WORD_INDEX(_bit_index) \
-    ((_bit_index) / UCS_BITMAP_BITS_IN_WORD)
+#define UCS_BITMAP_WORD_INDEX(_bitmap, _bit_index) \
+    _ucs_bitmap_word_index(_UCS_BITMAP_NUM_WORDS(_bitmap), (_bit_index))
+
+
+static inline size_t
+_ucs_bitmap_word_index(size_t bitmap_words, size_t bit_index)
```

Comment:
static UCS_F_ALWAYS_INLINE ucs_bitmap_word_index

Reply:
pls remove leading _ from function name

Reply:
Are you sure?

It is an internal function, not to be called from outside. I'd prefer the `UCS_BITMAP_WORD_INDEX` macro to be used by users.

Reply:
pls see https://github.com/openucx/ucx/blob/master/docs/CodeStyle.md#naming-convention